src/index.ts

```diff
@@ -33,7 +33,8 @@ export {
   MaxKey,
   MinKey,
   ObjectId,
-  Timestamp
+  Timestamp,
+  UUID
```

Comment:
This seems like an oversight, now that we have an API that advertises this as a return type, it seems more worth it than before that this is available as a top-level export